v3/src/models/codap/create-codap-document.ts

```diff
@@ -43,7 +44,8 @@ export function createCodapDocument(snapshot?: ICodapDocumentModelSnapshot, opti
       document.content?.addSharedModel(globalValueManager)
     }
     // Add the global value manager to the formula manager
-    getFormulaManager(document)?.addGlobalValueManager(globalValueManager)
+    const formulaManager = getFormulaManager(document) as Maybe<FormulaManager>
```

Comment:
Cast shouldn't be required because it matches return value of `getFormulaManager()`.

Reply:
It is required because `getFormulaManager` returns a `IFormulaManager` which doesn't include `addGlobalValueManager`. That is setup that way so the global value manager doesn't have to be included in the CODAP "library.